tests/ha/test_cluster_shutdown_start.py

```diff
@@ -551,10 +553,11 @@ def test_partial_mpu_bfr_aftr_clstr_rstrt_29474(self):
         LOGGER.info("Step 8: Successfully downloaded the object and verified the checksum")
 
         LOGGER.info("Step 9: Create multiple buckets and run IOs")
-        resp = self.ha_obj.perform_ios_ops(prefix_data='TEST-29474', nusers=1, nbuckets=10)
-        assert_utils.assert_true(resp[0], resp[1])
-        LOGGER.info("Cleaning up accounts and buckets created in IO operations")
-        resp = self.ha_obj.delete_s3_acc_buckets_objects(resp[2])
+        users = self.mgnt_ops.create_account_users(nusers=1)
+        self.test_prefix = 'test-29474'
+        self.s3_clean.update(users)
+        resp = self.ha_obj.ha_s3_workload_operation(s3userinfo=list(users.values())[0],
+                                                    log_prefix=self.test_prefix, nsamples=10)
```

Comment:
same here. Need to add skipcleanup=True

Reply:
No that was intentionally removed as we do not want to do delete when we are doing the test for multipart to seperate out issues